code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -418,6 +418,17 @@ var/list/ai_list = list()
 				H.attack_ai(src) //may as well recycle
 			else
 				src << "<span class='notice'>Unable to locate the holopad.</span>"
+		return
+
+	if(href_list["remotedoor"])
+		var/obj/machinery/door/airlock/A = locate(href_list["remotedoor"])

```

Comment:
Yes, lets allow the AI to open **all** doors with a few topic calls, what could go wrong?